grammar.js

```diff
@@ -522,6 +534,14 @@ module.exports = grammar({
       choice($.identifier, $.keyword_identifier, $.subscript, $.attribute)
     ),
 
+    // Extended patterns (patterns allowed in match statement are far more flexible than simple patterns
+
+    as_pattern: $ => prec.left(seq(
```

Comment:
or call this alias_expression?

Reply:
The patterns in a match case are a subset of expressions. I simplified using expression here